lib/features/tabs/ui/screens/components/top_picks_section_header.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:elvan/shared/components/constants/app_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class TopPicksSectionHeader extends StatelessWidget {
+  const TopPicksSectionHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: const [
+        Text(
```

Comment:
Use A Shared Text Widget like AppText where you get to pass style or have predefined styles.